Log message:
asm/backlight.h is ppc-only

Signed-off-by: Al Viro <viro@zeniv.linux.org.uk>

Modified files:
    linux/drivers/macintosh:
	adbhid.c 


Index: linux/drivers/macintosh/adbhid.c
Stats: 1 insertion, 1 deletion
http://linux-m68k-cvs.ubb.ca/c/cvsweb/linux/drivers/macintosh/adbhid%2ec.diff?r1=1.1.1.23&r2=1.9
========================================================================
--- linux/drivers/macintosh/adbhid.c	23 Sep 2006 16:43:55 -0000	1.1.1.23
+++ linux/drivers/macintosh/adbhid.c	27 Sep 2006 23:14:35 -0000	1.9
@@ -45,8 +45,8 @@
 #include <linux/pmu.h>
 
 #include <asm/machdep.h>
-#include <asm/backlight.h>
 #ifdef CONFIG_PPC_PMAC
+#include <asm/backlight.h>
 #include <asm/pmac_feature.h>
 #endif
 
-
To unsubscribe from this list: send the line "unsubscribe linux-m68k-cvscommit" in
the body of a message to majordomo@vger.kernel.org
More majordomo info at  http://vger.kernel.org/majordomo-info.html

